droid-hal-device.inc

```diff
@@ -320,6 +320,10 @@ Summary: Straggler files for %{device} hw
 
 %package kernel-dtbo
 Summary: Placeholder for %{device} flashable dtbo if applicable
+Requires: oneshot
+Requires(post): droid-config-flashing
+%{_oneshot_requires_post}
+
```

Comment:
nitpick: diff looks neater, if clean up is done first, like so: https://github.com/sledges/droid-hal-device/commits/more-dtbo

Otherwise LGTM!

Reply:
Make sense, updated PR.